apps/arweave/src/ar_block_pre_validator.erl

```diff
@@ -510,17 +566,29 @@ pre_validate_cumulative_difficulty(B, PrevB, SolutionResigned, Peer) ->
 					gen_server:cast(?MODULE, {enqueue, {B, PrevB, true, Peer}}),
 					enqueued;
 				false ->
-					pre_validate_quick_pow(B, PrevB, false, Peer)
+					pre_validate_packing_difficulty(B, PrevB, false, Peer)
```

Comment:
Is this ever called with `SolutionResigned == true`?

Reply:
Oh, good catch! This is wrong, fixed